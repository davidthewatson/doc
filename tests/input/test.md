# Test Title

This is a test paragraph with **bold text** and a [link](https://example.com).

> This is a blockquote.

- Item 1
- Item 2
- Item 3

