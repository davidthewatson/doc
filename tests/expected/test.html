<!doctype html>
<html lang="en" data-theme="light">
<head>
  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <title>Test Title</title>
  <link rel="stylesheet" href="/css/pico.min.css">
</head>
<body>
  <header class="container">
    <nav> <ul> 
    <li class="brand"> 
    <a href="/">davidwatson.org</a>
    </li> 
    </ul> 
    <ul>
    <li><a href="/media/chameleon/chameleon.pdf">resume</a></li>
    </ul>
    </nav> 
  </header>

  <main class="container">
    <h1>Test Title</h1>
    <p>This is a test paragraph with <strong>bold text</strong> and a <a href="https://example.com">link</a>.</p>
    <blockquote>This is a blockquote.</blockquote>
    <ul>
      <li>Item 1</li>
      <li>Item 2</li>
      <li>Item 3</li>
    </ul>
  </main>

  <footer>
    <nav class="container">
      <ul>
        <li><a href="https://www.linkedin.com/in/davidthewatson/" alt="LinkedIn" title="LinkedIn"><i class="fa fa-brands fa-linkedin fa-3x"></i></a></li>
        <li><a href="/about/"><img class="icon-rounded fa fa-brands fa-linkedin fa-3x" src="/images/watson.jpg" alt="Dave" title="Dave"></a></li>
        <li><a href="https://github.com/davidthewatson" alt="Github" title="Github"><i class="fa fa-brands fa-github fa-3x"></i></a></li>
      </ul>
    </nav>
    <nav class="container">
      <ul>
        <li>Â© <a href="/about/">David Watson</a> 2024</li>
      </ul>
      <ul>
        <li><a href="/about/">about</a></li>
      </ul>
    </nav>
  </footer>
</body>
</html>

